packages/tools-api/src/domain/encoding.ts

```diff
@@ -30,7 +30,27 @@ export function decodeType(type: string, encoded: `0x${string}`): AbiValue {
   return decodeParsed(parseType(type), encoded)
 }
 
+const DEBUG = false
```

Comment:
Either leftover or extract it to some higher place that allows every file to see it/use it